Dockerfile

```diff
@@ -25,13 +25,12 @@ RUN apt-get update && \
 FROM base AS app-build
 
 # Add project files
-ADD CMakeLists.txt /src/
-ADD config.h.in /src/
-ADD /cmake/ /src/cmake
-ADD /dep/ /src/dep
-ADD /src/ /src/src
-ADD /utils/ /src/utils
-ADD /.git/ /src/.git
```

Comment:
pretty sure the `.git` isn't required so I just dropped it.

Reply:
I found out that it is actually used by cmake to read a version. I should restore the .git folder, or find a better solution for version